tests/system/action/user/test_update.py

```diff
@@ -190,6 +199,16 @@ def test_committee_manager_without_committee_ids(self) -> None:
         )
         self.assertCountEqual(user["committee_ids"], [60, 61])
         self.assertCountEqual(user["committee_$can_manage_management_level"], [60, 61])
+        self.assert_history_information(
+            "user/111",
+            [
+                "Personal data changed",
+                "Participant removed from group {} in meeting {}",
+                "group/600",
+                "meeting/600",
+                "Committee Management Level changed",
```

Comment:
Why are the group and the meeting not insert into the Part removed message? it looks broken.